protocol/communities/manager.go

```diff
@@ -2386,9 +2386,6 @@ func (m *Manager) HandleCommunityCancelRequestToJoin(signer *ecdsa.PublicKey, re
 	if err != nil {
 		return nil, err
 	}
-	if community == nil {
-		return nil, ErrOrgNotFound
-	}
```

Comment:
Since `GetByID` has been reintroduced to this function, this check should also be included.